docs/source/telemetry.mdx

```diff
@@ -0,0 +1,189 @@
+
+**🚀 5-minute setup:**
+1. **Start local observability stack**: `docker run -p 3000:3000 -p 4317:4317 -p 4318:4318 --rm -ti grafana/otel-lgtm`
+2. **Add telemetry config** to your `config.yaml`:
+   ```yaml
+   telemetry:
+     exporters:
+       metrics:
+         otlp:
+           endpoint: "http://localhost:4318/v1/metrics"
+           protocol: "http/protobuf"
+       tracing:
+         otlp:
+           endpoint: "http://localhost:4318/v1/traces" 
+           protocol: "http/protobuf"
+   ```
+3. **Restart your MCP server** with the updated config
+4. **Open Grafana** at http://localhost:3000 (admin/admin) and explore your telemetry data
+
+For detailed steps and dashboard examples, see the [complete Grafana setup guide](guides/telemetry-grafana.mdx).
+
+### **Production Deployment**
+
+For production environments, configure your MCP server to send telemetry to any OTLP-compatible backend. The Apollo MCP Server uses standard OpenTelemetry protocols, ensuring compatibility with all major observability platforms.
+
+#### **Configuration Example**
+
+```yaml
+telemetry:
+  service_name: "mcp-server-prod"      # Custom service name
+  exporters:
+    metrics:
+      otlp:
+        endpoint: "https://your-metrics-endpoint"
+        protocol: "http/protobuf"       # or "grpc"
+    tracing:
+      otlp:
+        endpoint: "https://your-traces-endpoint" 
+        protocol: "http/protobuf"
+```
+
+#### **Observability Platform Integration**
+
+The MCP server works with any OTLP-compatible backend. Consult your provider's documentation for specific endpoint URLs and authentication:
+
+- **[Datadog OTLP Integration](https://docs.datadoghq.com/tracing/setup_overview/open_standards/otlp_ingest_in_datadog/)** - Native OTLP support
+- **[New Relic OpenTelemetry](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/)** - Direct OTLP ingestion
+- **[AWS Observability](https://aws-otel.github.io/docs/introduction)** - Via AWS Distro for OpenTelemetry
+- **[Grafana Cloud](https://grafana.com/docs/grafana-cloud/send-data/otlp/)** - Hosted Grafana with OTLP
+- **[Honeycomb](https://docs.honeycomb.io/getting-data-in/opentelemetry/)** - OpenTelemetry-native platform
+- **[Jaeger](https://www.jaegertracing.io/docs/1.50/deployment/)** - Self-hosted tracing
+- **[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/deployment/)** - Self-hosted with flexible routing
+
+#### **Production Configuration Best Practices**
+
+**Environment and Security:**
+```yaml
+# Set via environment variable
+export ENVIRONMENT=production
+
+telemetry:
+  service_name: "apollo-mcp-server"
+  version: "1.0.0"                     # Version for correlation
+  exporters:
+    metrics:
+      otlp:
+        endpoint: "https://secure-endpoint"  # Always use HTTPS
+        protocol: "http/protobuf"           # Generally more reliable than gRPC
+```
```

Comment:
I think it would make sense to either introduce or explain this. I don't know which is better, but I feel like we need to explain what this file does for the environment and security settings.